Decred Wallet/Assets.xcassets/ic_checkmark_round.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "ic_checkmark_64px.pdf"
```

Comment:
Line 55 of `SeedBackupReminderViewController.swift` also needs to be corrected to use this icon instead of `ic_checkmark`.